README.md

```diff
@@ -26,6 +26,19 @@ For example, publishers may wish to add statistics on minority or women-owned bu
 
 The codelist's Category column indicates whether the statistic applies to bids or bidders or whether it is specified or required by a particular regulatory context (e.g. EU).
 
+## Guidance
+
+### Correcting a bid value
+
+Procuring entities and, in some jurisdictions, bidders, can correct bid values after a bid is submitted; for example, to correct a purely arithmetic error or a misplaced decimal.
+
+In OCDS:
+
+* The bid value as submitted goes in `bids.details.value`
+* If a correction is made, that bid value is overwritten
```

Comment:
The first bullet makes it sound like the field is *always* equal to the bid value as submitted. But that's only true at the time of submission.

Also, avoid metaphorical language like "goes in". (Can add this content to the standard handbook: https://ocp-software-handbook.readthedocs.io/en/latest/general/documentation.html#word-choice)

```suggestion
In OCDS, the bid's value is disclosed via the `bids.details.value` field. If a bid's value is corrected, the value of the `bids.details.value` field is overwritten. As such, the originally submitted value is only available via the contracting process' [change history](https://standard.open-contracting.org/latest/en/primer/releases_and_records/).
```